command.go

```diff
@@ -120,7 +121,7 @@ func (c *cancellableCommand) Stop(gracePeriod time.Duration) {
 }
 
 // NewCommand returns a new command based on passed CommandInfo.
-func NewCommand(commandInfo mesos.CommandInfo, env []string, scr scraper.Scraper) (Command, error) {
+func NewCommand(commandInfo mesos.CommandInfo, env []string, scr scraper.Scraper, appender appender.Appender) (Command, error) {
```

Comment:
instead of providing `scr scraper.Scraper, appender appender.Appender` I'd prefer to pass a function that handles `stdout, stderr io.Writer`

Reply:
Can you provide a example code for this?

Reply:
Yes, I was a bit worried about too much arguments to this function, but didn't see a better way. It would be great to do this like @janisz suggests.

Reply:
I will provide configuration options in the same way as it is in Logstash appender.

Reply:
@janisz @AlfredBroda done.